extension/src/background/ducks/session.ts

```diff
@@ -57,6 +60,25 @@ export const sessionSlice = createSlice({
         privateKey: "",
       };
     },
+    setPassword: (state, action: { payload: AppData }) => {
+      const { password } = action.payload;
+
+      return {
+        ...state,
+        password,
+      };
+    },
```

Comment:
in order to prevent the user from having to re-enter their password, we can store the password in the background in memory for use later in the flow. 

This actually makes this [issue](https://github.com/stellar/freighter/issues/903) possible, so I thought it made sense to introduce this feature here